docs/guide/ecosystem/server-side-rendering.md

```diff
@@ -1,22 +1,110 @@
+
+### SEO friendly 
+
+Most crawlers cannot scan websites that are rendered with JavaScript, resulting search engines registering empty pages without content. This directly leads to bad search engine ranking. 
+
+With SSR, pages are returned as fully rendered HTML by the server, allowing for crawlers to scan at will.
+
+### Fast load on slower devices
+
+Applications with allot of initial rendering, may cause slower devices to struggle. This is a real problem, as the general population is not carrying cutting edge mobile devices. 
+
+With SSR, the page is fully rendered on the server, eliminating that initial burden for lower end devices. When paired with some other optimisation techniques, like code splitting, pages can load marginally faster.
+
+### Social Presence
+
+When sharing your website on various social media, they will show a small preview and extract metadata from it. Normal SPA applications are very poor at this and often result in blank pages with little to no metadata. Social media crawlers cant really parse JavaScript, so they dont know what to do with SPAs.
+
+With SSR, pages are displayed with nice previews, as both the page content it self and meta tag data is present, allowing for social media crawlers to extract what they need.
+
+### Faster initial interaction
+
+Even though the browser still needs to download and run the whole JavaScript bundle, because the page is already rendered, the user can actually start browsing at that point. This can directly result in a lower drop-off ratio, as users hate waiting.
+
+### Removes white page flicker
+
+This is not as big of a deal, but because the page does not need to be rendered on the client, there will not be a white flicker while the page is being being rendered. This can however be easily mitigated with a loading screen or something similar. 
+
+## Problems with SSR
+
+Even though SSR sounds awesome, there are some things that need to be taken in consideration.
+
+### Time to interaction
+
+Even though they can see the website, it does'nt meant it is working. For very dynamic websites, with allot of JavaScript logic driving the UI, this can lead to weird situations where the page is rendered, looks ready, but the app bundle is still downloading, so no JavaScript logic can be executed yet.
+
+### Time to first bite (TTFB) is slower
+
+Because the server has to actually do the rendering, fetch async data and so on, the time it takes for the response to hit the browser is bigger.
+
+### Server costs
+
+Because there is usually a live Node.js instance always on, waiting to render a request, server costs are higher than static file hosting.
+
+Check the [Hosting page](./hosting.md#server-rendered-websites) for more info.
+
+### Server Load
+
+This is directly related to the above. Server load is higher while pages are being rendered, sometimes blocking other operations until the process is done.
+
+### Window not available
```

Comment:
Not only `window`, [platform-specific APIs](https://blog.lichter.io/posts/the-guide-to-write-universal-ssr-ready-vue-compon/#window%2C-document%2C-and-friends---platform-specific-apis).

Mentioning where it shouldn't be called would be a nice thing as well.